pallets/liquid-staking/src/tests.rs

```diff
@@ -219,20 +219,71 @@ fn test_transact_bond_extra_work() {
 
     ParaA::execute_with(|| {
         assert_ok!(LiquidStaking::bond(
-            Origin::signed(ALICE),
             ALICE,
             2 * DOT_DECIMAL,
             RewardDestination::Staked
         ));
 
-        assert_ok!(LiquidStaking::bond_extra(
-            Origin::signed(ALICE),
-            3 * DOT_DECIMAL
-        ));
+        assert_ok!(LiquidStaking::bond_extra(3 * DOT_DECIMAL));
     });
 
     Relay::execute_with(|| {
         let ledger = RelayStaking::ledger(ALICE).unwrap();
         assert_eq!(ledger.total, 5 * DOT_DECIMAL);
     });
 }
+
+#[test]
+fn test_transact_unbond_work() {
+    TestNet::reset();
+
+    ParaA::execute_with(|| {
+        assert_ok!(LiquidStaking::bond(
```

Comment:
Cloud you please use the case that Bond 5 and then Unbond 2? And ensure ledger.total = 5, ledger.active = 3.